### PR TITLE
[MT-4449] Autoformat

### PR DESCRIPTION
**Pull request by aspirisen**
Ticket: https://linear.app/prezly/issue/MT-4449/editor-detect-ordered-and-unordered-lists-headings-and-quotes

Demo:
Autoformat:

Undo:

### CODE REVIEW

File: packages/slate-editor/src/modules/editor-v4-autoformat/autoformatUtils.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import type { Editor } from 'slate';
+import type { PlateEditor } from '@udecode/plate-core';
+import { getParent, isElement, isType } from '@udecode/plate-core';
+import { unwrapList, toggleList } from '@udecode/plate-list';
```
Comment (e1himself) [inline packages/slate-editor/src/modules/editor-v4-autoformat/autoformatUtils.ts:4]: Hm... We have our own lists plugin :thinking:

**Comment by digitalbase** [comment]
hi @aspirisen what is the state of this PR?

**Review by e1himself — APPROVED** [review]

**Comment by aspirisen** [comment]
@digitalbase going to be merged

**Pull request merged**